src/models/trips.go

```diff
@@ -0,0 +1,111 @@
+package models
+
+import (
+	"github.com/Nastya-Kruglikova/cool_tasks/src/database"
+	"github.com/satori/go.uuid"
+)
+
+const (
+	createTrip        = "INSERT INTO trips (user_id) VALUES ($1) RETURNING trip_id;"
+	getTripIDByUserID = "SELECT trips.trip_id FROM trips WHERE trips.user_id = $1;"
+	getTripsByTripID  = "SELECT trips.user_id FROM trips WHERE trip_id = $1;"
+)
+
+type Trip struct {
+	TripID      uuid.UUID
+	UserID      uuid.UUID
+	Events      []Event
+	Flights     []Flight
+	Museums     []Museum
+	Restaurants []Restaurant
+	Hotels      []Hotel
+	Trains      []Train
+}
+
+var CreateTrip = func(trip Trip) (uuid.UUID, error) {
+	var id uuid.UUID
+	err := database.DB.QueryRow(createTrip, trip.UserID).Scan(&id)
+
+	return id, err
+}
+
+var GetTripsByTripID = func(id uuid.UUID) (Trip, error) {
+
+	var (
+		trip        Trip
+		err         error
+		events      []Event
+		flights     []Flight
+		museums     []Museum
+		hotels      []Hotel
+		trains      []Train
+		restaurants []Restaurant
+	)
+
+	trip.TripID = id
+
+	events, err = GetEventsByTrip(id)
```

Comment:
If you write something like this :
trip.Events, err = GetEventsByTrip(id) , 
a variable "events" will not be needed.